test-cases/longevity/longevity-lwt-500G-3d.yaml

```diff
@@ -0,0 +1,33 @@
+# Test duration 15 hours temporary. Should be 3 days. Will be changed later
+test_duration: 900
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50" ]
```

Comment:
How long does it take to populate?
What is the load during populate?

Reply:
It's pretty long. I am working on it to improve the time

Reply:
With i3.4xlarge db node and c5.2xlarge loaders it takes about 9 hours